app/models/order.js

```diff
@@ -37,5 +38,15 @@ export default ModelBase.extend(CustomPrimaryKeyMixin, {
   discountCode     : belongsTo('discount-code'),
   accessCode       : belongsTo('access-code'),
   tickets          : hasMany('ticket', { readOnly: true }),
-  attendees        : hasMany('attendee')
+  attendees        : hasMany('attendee'),
+
+  taxAmount: computed('amount', 'event.tax.isTaxIncludedInPrice', 'event.tax.rate', function() {
+    const taxType = this.event.get('tax.isTaxIncludedInPrice');
+    const taxRate = this.event.get('tax.rate');
+    if (taxType) {
+      return ((taxRate * this.amount) / (100 + taxRate)).toFixed(2);
+    } else {
+      return ((this.amount) / (1 + taxRate / 100) * (taxRate / 100)).toFixed(2);
```

Comment:
Can you share thoght process behind this.

Reply:
Let x be amount without taxIncluded
So we need to calculate tax which will can be calculated by totalAmount = x + x*taxRate/100;
where ```x * taxRate / 100``` is tax added on top of it which is given by 
```js
totalAmount / ( 1 + taxRate / 100 ) * ( taxRate / 100 )  which gives
(taxRate * totalAmount) / (100 + taxRate)
```